docs/source/environments/basalt.rst

```diff
@@ -0,0 +1,16 @@
+MineRL BASALT Competition Environments
+=======================================
+*The videos on this page do not represent the new MineRL environment or the new dataset.*
```

Comment:
Nit but this could be a warning tag

Reply:
Tried to clean up thsi part but decided to remove it: it is bit too early in the docs to start throwing notes/warnings. They still serve as a nice visual abstract 🙃